float64 time
float64 tau
float64 dotq
float64 q
float64 dotp_EE
float64 terminal_energy
float64 sphere
float64 cylinder
float64 wall